src/broadcast.jl

```diff
@@ -32,6 +32,7 @@ broadcast_ops = [
     # "fdim",
     ("invxback","invxback","(-xi*yi*yi)"),
     ("reluback","reluback","(yi>0?xi:0)"),
+    ("eluback", "eluback", "ifelse(yi>0,dyi,yi+1)"),
```

Comment:
This part turns to a cuda code snippet. So, I think it must be replaced with the following:
```
("eluback", "eluback", "(yi>0?xi:yi+1)")
```